src/select/select.tsx

```diff
@@ -73,23 +86,36 @@ class Select<T> extends React.Component<Props<T>, State> {
 
     private getOptionsList() {
         return (
-            <ul className="select-option-items" data-qa={`select--option-items-${this.props.id}`}>
-                {this.props.options.map((option, index) => {
-                    const selectedClass = this.isSelected(option) ? 'selected ' : '';
-                    return (
-                        <li
-                            key={index}
-                            role="option"
-                            onClick={this.handleOptionClick.bind(this, option.value)}
-                            data-qa={`select--option-${option.label}`}
-                            className={`item ${selectedClass}`}
-                            aria-selected={this.isSelected(option)}
-                        >
-                            {option.label}
-                        </li>
-                    );
-                })}
-            </ul>
+            <>
+                {this.props.searchable && <div className="select-filter-bar">
+                    <input
+                        type="text"
+                        data-qa={`select--filter-${this.props.id}`}
+                        className="form-control"
+                        placeholder="search"
```

Comment:
Please make it configurable via Props. Otherwise there is no way of providing i18n.

Reply:
Ok, added another optional parameter called `searchPlaceholder`.